src/helpers/renderReact.tsx

```diff
@@ -0,0 +1,45 @@
+import Badge from '@/components/Badge'
+import QRCodeLabel from '@/components/icons/QRCodeLabel'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+import htmlTemplate from '@/helpers/htmlTemplate'
+import type { Options } from 'qr-code-styling'
+
+const baseQrConfig = {
+  margin: 10,
+  width: 485,
+  height: 485,
+  type: 'canvas',
+
+  dotsOptions: {
+    color: '#fed823',
+    type: 'dots',
+  },
+  cornersSquareOptions: {
+    color: '#ff7bed',
+    type: 'dot',
+  },
+  cornersDotOptions: {
+    type: 'dot',
+  },
+  backgroundOptions: {
+    color: '#0d0030',
+  },
+} as Options
+
+export default function (
+  tokenAddress: string,
+  tokenId: string,
+  derivativeName: string
+) {
+  const html = ReactDOMServer.renderToStaticMarkup(
+    <Badge derivativeName={derivativeName} />
+  )
+  const config = {
+    ...baseQrConfig,
```

Comment:
Why do we need to separate those configs?

Reply:
I thought that it'll be easier to read. Fixed it.